.travis/setup

```diff
@@ -5,28 +5,14 @@ if [ -z "$INSTALL4J_LICENSE" ]; then
  exit -1
 fi
 
-export WINDOWS_32_JRE="windows-x86-1.8.0_66.tar.gz"
-export WINDOWS_64_JRE="windows-amd64-1.8.0_66.tar.gz"
-export MAC_JRE="macosx-amd64-1.8.0_66.tar.gz"
 export ASSET_URL="http://github.com/triplea-game/assets/blob/master/install4j"
```

Comment:
My mistake, you are correct, it is still used. It doesn't need the "export", but we do need the var.